ui/src/components/Issuer/IssueAsset.tsx

```diff
@@ -21,15 +27,27 @@ const IssueAsset = () => {
     const [ quantityPrecision, setQuantityPrecision ] = useState<string>('')
     const [ description, setDescription ] = useState<string>('')
     const [ isPublic, setIsPublic ] = useState<boolean>(true)
-    const [ observersList, setObserversList ] = useState<string>('');
+    const [ observers, setObservers ] = useState<string[]>([]);
     const [ error, setError ] = useState<ErrorMessage>();
     const [ loading, setLoading ] = useState(false);
 
+    const allRegisteredParties = [
+            useStreamQueryAsPublic(RegisteredCustodian).contracts
+                .map(ri => ({ contractId: ri.contractId, contractData: ri.payload.custodian })),
```

Comment:
super minor but I presume that `ri` was an abbreviation for `RegisteredInvestor`? perhaps we can rename the variables